src/content/docs/pt-br/pages/servicos/technical-support.mdx

```diff
@@ -37,12 +35,11 @@ O usuário deve estar previamente autorizado pelo cliente e configurado na ferra
 
 | Plano de Serviço | Canais de Suporte Técnico |
 | ---------------- | ------------------------- |
-| Developer | Acesso ilimitado a documentação, guias, whitepapers e relatórios de bugs. |
 | Business | Suporte 24 horas por dia, 7 dias por semana, via ticket e email. |
-| Enterprise | Suporte 24/7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
-| Mission Critical | Suporte 24/7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
+| Enterprise | Suporte 24×7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
```

Comment:
If we changed 24/7 by 24x7, shouldn't we change 8/5 by 8x5?

Reply:
Done!